.github/workflows/ci-static-build-test.yml

```diff
@@ -1,12 +1,13 @@
 name: "static-build-test-p4c"
 
 on:
+  schedule:
+    # Every day on midnight UTC
+    - cron: "0 0 * * *"
   push:
     branches: [main]
   pull_request:
     branches: [main]
-  merge_group:
-    branches: [main]
```

Comment:
@fruffy Why is this not present in the scheduled builds? I've checked the existing and none contains it.

Reply:
What do you mean?

Reply:
The `on`->`merge_group` key is not present in any of the CI jobs that have `schedule`.